lib/view_model/wallet_restore_view_model.dart

```diff
@@ -183,6 +191,17 @@ abstract class WalletRestoreViewModelBase extends WalletCreationVM with Store {
             language: 'English',
           );
 
+      case WalletType.zano:
+        return zano!.createZanoRestoreWalletFromKeysCredentials(
```

Comment:
@OmarHatem28 nope we don't have it, we never had it actually in Zano because seed phrase is more practical(it stores the account creation date, so the scan started from right timestamp). I believe since we never had it and no users ever used it as backup it's ok to not have it in the wallet.